src/test/java/seedu/weme/logic/commands/CommandTestUtil.java

```diff
@@ -103,11 +101,13 @@ public static void assertCommandFailure(Command command, Model actualModel, Stri
     public static void showMemeAtIndex(Model model, Index targetIndex) {
         assertTrue(targetIndex.getZeroBased() < model.getFilteredMemeList().size());
 
+        /*
         Meme meme = model.getFilteredMemeList().get(targetIndex.getZeroBased());
-        final String[] splitName = meme.getName().fullName.split("\\s+");
-        model.updateFilteredMemeList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
+        final List<ImageUrl> splitName = new ArrayList<>(meme.getUrl());
+        model.updateFilteredMemeList(new TagContainsKeywordsPredicate(Arrays.asList(splitName.get(0).value)));
 
         assertEquals(1, model.getFilteredMemeList().size());
+         */
```

Comment:
Can consider creating a PathContainsKeyword class to work with showMemeAtIndex to replace the original NameContainsKeywords

Reply:
Ok got it.